src/utils/DragAndDrop.js

```diff
@@ -181,6 +182,68 @@ define(function (require, exports, module) {
             var pathList = [];
             var errorList = [];
 
+            function handleRegularFile(deferred, file, filename, buffer, encoding) {
+                file.write(buffer, {encoding: encoding}, function(err) {
+                    if (err) {
+                        deferred.reject(err);
+                        return;
+                    }
+
+                    if(encoding === "utf8") {
```

Comment:
We just won't automatically open them, but they'll be in the tree.  If you want me to do a check for any image type, and override, I could.  How much does it matter?